Makefile

```diff
@@ -24,8 +23,11 @@ fioctl-%:
 	@test -x $(builder) || (echo "Please install xgo toolchain $(HOME)/go/bin: go install github.com/crazy-max/xgo@v0.30.0")
 	$(eval GOOS:=$(shell echo $* | cut -f1 -d\- ))
 	$(eval GOARCH:=$(shell echo $* | cut -f2- -d\-))
+	$(eval TARGET_GOTAGS:=netgo,osusergo)
+	$(eval TARGET_LDFLAGS:=$(if $(shell test $(GOOS) = linux && echo "ok"),'-extldflags=-static',))
+	$(eval TARGET_LDFLAGS:=$(if $(shell test $(GOOS) = windows && echo "ok"),'-extldflags=-static',$(TARGET_LDFLAGS)))
```

Comment:
just to make sure I'm groking this: These eval statements put these variable into the environment so that xgo also gets them, correct?

Reply:
No, this is a weird Makefile syntax to set the variable inside the target.
GNU Make ignores all such variables, unless wrapped into the eval.

Anyway, I'm not quite fond of this script and will try to split into separate targets.
I'm really really scared that this Makefile becomes not only hard to grok, but may also fail on other platforms than GNU make.